src/App.js

```diff
@@ -65,10 +65,13 @@ export default function App() {
     async function getRequest() {
         const res = await fetch(`https://cjsback.herokuapp.com/`)
         const data = await res.json()
+        return data
```

Comment:
felt that it was strange to not have get request return anything even though I am currently not doing anything with data

Reply:
Suggestion: As a general rule of thumb only expose only returns if it used or will be used immediately. Helps makes the code less confusing for posterity if the return is unused. Based on that I would remove the return statement.

Also a better name might be wakeUpServer for this function to highlight without having to comment that this is used exclusively to wake up the server.

Reply:
+1 on wakeUpServer

Reply:
got it!